tests/plotting/test_renderer.py

```diff
@@ -177,3 +178,11 @@ def test_legend_face(sphere, face):
     pl = pv.Plotter()
     pl.add_mesh(sphere, label='sphere')
     pl.add_legend(face=face)
+
+
+def test_add_legend_background_opacity(sphere):
+    background_opacity = default_rng(0).random()
```

Comment:
Small point: Is there a reason not to hard code a value here?  Seeding a RNG will result in the same value each time as well, we might as well make it explicit for easier human understanding.

Reply:
Only my mistake. I want to test each time with a random value between 0 and 1. Thank you for catching it.

```suggestion
    background_opacity = default_rng().random()
```

Reply:
I see, that is easier to understand.